### PR TITLE
Fix bridge support flyover tests after locking cap refactor

### PR DESCRIPTION
**Pull request by nathanieliov**
- Fix registerBtcTransaction_after_RSKIP134_activation_sends_above_lockingcap test
- Fix BridgeSupportFlyoverTest after LockingCap refactor

## Motivation and Context

Since the locking cap refactor `BridgeSupportFlyoverTest` has been failing, those tests needed to be fixed once the refactor was done.

## How Has This Been Tested?

Unit testes

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

### CODE REVIEW

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java [hunk 1]
```diff
@@ -17,37 +17,85 @@
...
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
```
Comment (marcos-iov) [inline rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java:37]: Please rollback, it's preferable to use wildcards. Makes it cleaner

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java [hunk 2]
```diff
@@ -85,6 +117,7 @@ class BridgeSupportFlyoverTest {
     private FederationSupportBuilder federationSupportBuilder;
     private ActivationConfig.ForBlock activations;
     private SignatureCache signatureCache;
+    private LockingCapSupport lockingCapSupport;
```
Comment (julia-zack) [inline rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java:120]: nit: i'd move the variable declaration to above, right after the `whitelistSupport` one

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java [hunk 3]
```diff
@@ -95,6 +128,7 @@ void setUpOnEachTest() {
         bridgeSupportBuilder = new BridgeSupportBuilder();
         whitelistSupport = mock(WhitelistSupport.class);
         federationSupportBuilder = new FederationSupportBuilder();
+        lockingCapSupport = mock(LockingCapSupport.class);
```
Comment (julia-zack) [inline rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java:131]: same here

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java [hunk 4]
```diff
@@ -2093,13 +2129,17 @@ void registerFlyoverBtcTransaction_failed_when_tx_with_witness_surpassing_lockin
...
+        when(lockingCapSupport.getLockingCap()).thenReturn(Optional.of(lockingCapValue));
+
         FederationSupport federationSupport = federationSupportBuilder
             .withFederationConstants(bridgeConstants.getFederationConstants())
             .withFederationStorageProvider(federationStorageProvider)
             .withActivations(activations)
             .build();
 
         BridgeSupport bridgeSupport = bridgeSupportBuilder
+            .withLockingCapSupport(lockingCapSupport)
```
Comment (julia-zack) [inline rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java:2142]: same here, move it closer to `[here](https://github.com/rsksmart/rskj/pull/2663/files#diff-e595294bfc4b99fd081ffaf7c754cdf9b64bd6b9c8494f71d6ddaf957bf8045bR2150)`

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java [hunk 5]
```diff
@@ -2502,6 +2546,7 @@ void registerFlyoverBtcTransaction_failed_when_tx_without_witness_surpassing_loc
...
```
Comment (julia-zack) [inline rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java:2549]: same here :)

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java (hunk 2 above)
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java:120]: Done

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java (hunk 1 above)
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java:37]: Done

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java (hunk 3 above)
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java:131]: Done

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java (hunk 4 above)
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java:2142]: Done

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java (hunk 5 above)
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java:2549]: Done

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java (hunk 1 above)
Comment (marcos-iov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java:37]: I don't see the change

**Review by julia-zack — APPROVED** [review]
lgtm :)

File: rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java (hunk 1 above)
Comment (nathanieliov) [reply rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java]: Sorry, the push failed. Check again please.

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**